lib/procore.ex

```diff
@@ -52,13 +52,14 @@ defmodule Procore do
   def send_request(
         %Request{
           request_type: :get,
+          api_version: api_version,
           endpoint: endpoint,
           query_params: query_params
         },
         client
       )
       when byte_size(endpoint) > 0 do
-    http_client().get(client, endpoint, QueryParams.build(query_params))
+    http_client().get(client, "#{api_version}#{endpoint}", QueryParams.build(query_params))
```

Comment:
I would create a function that normalizes the endpoint rather than repeating the string concat, easier to refactor in the future

```suggestion
    http_client().get(client, endpoint_url(api_version, endpoint), QueryParams.build(query_params))
```